tests/core/pyspec/eth2spec/test/helpers/deposits.py

```diff
@@ -62,29 +64,69 @@ def deposit_from_context(spec, deposit_data_list, index):
     return deposit, root, deposit_data_list
 
 
-def prepare_genesis_deposits(spec, genesis_validator_count, amount, signed=False, deposit_data_list=None):
+def prepare_full_genesis_deposits(spec,
+                                  amount,
+                                  pubkey_max_range,
+                                  pubkey_min_range=0,
+                                  signed=False,
+                                  deposit_data_list=None):
     if deposit_data_list is None:
         deposit_data_list = []
     genesis_deposits = []
-    for validator_index in range(genesis_validator_count):
+    for validator_index in range(pubkey_min_range, pubkey_max_range):
```

Comment:
Note that
- in `prepare_full_genesis_deposits`, `pubkey_max_range` is not included, i.e., `pubkey_min_range 
 <= value < pubkey_max_range`.
- in `prepare_random_genesis_deposits`, `pubkey_max_range` is included, i.e., `pubkey_min_range 
 <= value <= pubkey_max_range`.


It seems fine in the current test cases since we usually only pass `pubkey_max_range=deposit_count` to `prepare_full_genesis_deposits`. But it's better to rename the parameter names.

I'd suggest

1. renaming `prepare_full_genesis_deposits` parameters:
    - `pubkey_max_range` -> `deposit_count`
    - `pubkey_min_range` -> `min_validator_index`

2. and change this line to:
```suggestion
    for validator_index in range(min_validator_index, deposit_count + min_validator_index):
```

3. and change the caller from
    ```python
    prepare_full_genesis_deposits(
        ...
        pubkey_max_range=main_deposit_count + 1,
        pubkey_min_range=main_deposit_count,
        ...
    )
    ```
    to
    ```python
    prepare_full_genesis_deposits(
        ...
        deposit_count=1,
        min_validator_index=main_deposit_count,
        ...
    )
    ```